versions/1/data_stream/spec.yml

```diff
@@ -34,6 +34,22 @@ spec:
       name: elasticsearch
       additionalContents: false
       contents:
+      - description: Folder containing Elasticsearch Ilm Policy Definition
+        type: folder
+        name: ilm
+        additionalContents: false
+        contents:
+        - description: Supporting ilm policy definitions in YAML
```

Comment:
nit: please make sure you're using consistent abbrs in terms of capital letters (ilm vs ILM vs Ilm)